web/src/actions/seasons.js

```diff
@@ -56,6 +57,23 @@ export const fetchPreferences = seasonID => async dispatch => {
   }
 };
 
+export const setSwitchSeason = seasonID => async dispatch => {
+  try {
+    await postPreferences(seasonID);
+
+    dispatch({
+      type: FETCH_PREFERENCES_SUCCESS,
```

Comment:
the type "FETCH_PREFERENCES_SUCCES" don't mean a lot here
same for "FETCH_PREFERENCES_FAILED" below.